payu/models/cesm_cmeps.py

```diff
@@ -273,26 +287,44 @@ def _get_variable_span(self, section, variable):
             self.contents,
             re.DOTALL,
         )
```

Comment:
It's a relatively complex regular expression, it would be nice to have a an example of what it is suppose to match in a comment.

Also *cough* tests ...

Reply:
This isn't even part of this PR! But yes, you're right...

Reply:
Addressed in [9675ec8](https://github.com/payu-org/payu/pull/433/commits/9675ec8d6e28af60643b52ceb01e44a0bd21d40c)